pom.xml

```diff
@@ -359,6 +359,11 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
+      <id>avro-release-candidate</id>
+      <name>Avro Release Candidate</name>
+      <url>https://repository.apache.org/content/repositories/orgapacheavro-1037/</url>
```

Comment:
According to the Apache Avro dev mailing list, this seems to be the correct RC1 location. Could you trigger once more if you want to test Avro 1.12.0 RC, @Fokko ?
- https://lists.apache.org/thread/tqj0zk7qqsgr6tw10247grz3dv3svhtn

Reply:
To make CI happy, you need to update SBT file too.

https://github.com/apache/spark/blob/112a52d58816a842b4a2aa9f37367e1317623625/project/SparkBuild.scala#L272-L279

Reply:
Thanks @dongjoon-hyun for jumping in here. I wanted to test the RC with Spark to see if there are any breaking changes. I had it on my list to figure out how to fix SBT, but thanks for the pointer, that's helpful 👍